libs/scripts/release.sh

```diff
@@ -66,7 +75,7 @@ function git-reset () {
 function bump-version () {
     (
         # Patch the version
-        VERSION=$(npm version patch)
+        VERSION=$(npm version ${RELEASE_VERSION} --preid=${PREID})
```

Comment:
if not specified, can we have this do `npm version patch`?

Reply:
also if version is set, but no preid, what happens?

Reply:
if no preid, it reuses any existing preid (eg if the version in package.json has alpha, it'll stay alpha. If it's empty, it'll stay empty - the same as not including the `--preid=` option)

Reply:
> if not specified, can we have this do `npm version patch`?

@rickyrombo does this work by default too?

Reply:
yep! sorry, forgot to comment about it but I added a commit for it!

Reply:
sneaky i see it! awesome